src/Common/tests/DefaultOptionsProviderTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// ***********************************************************************
```

Comment:
I've disabled it for Silverlight. Do you think I should revert it for Silverlight?